### ISSUE TITLE
Mazumder - Perform 1 Genome Assembly in Geneious

### ISSUE THREAD

**Issue by JingyueWu** [FDA-ARGOS/data.argosdb#188]
1. Follow the tutorial and use Geneious to run genome assembly on SRR14460626 (B.1.351) 
2. Compare results with HIVE SPAdes, MEGAHIT and metaviralSPAdes on Galaxy
3. Talk with Dr. Mazumder and make a decision of 
A) if we should continue using the current available assemblers on other platforms (i.e. Galaxy/HIVE)
OR
B) if it’s worthwhile to purchase Geneous subscription for the lab

**Comment by steph-sing** [comment]
I don't think this task is complete, and there is not status update indicating completion.

**Comment by JingyueWu** [comment]
This task was completed last week, my apologies for the delayed feedback, please see below: 

First of all, from UX point of view, Geneious is very convenient and easy to navigate around. Its interface is a lot like R studio (See below). 

<img width="1419" alt="Screen Shot 2023-01-25 at 3 23 09 PM" src="https://user-images.githubusercontent.com/61590108/214681600-ffcceb2e-0fca-4778-8973-6b616104213b.png">

However, from the actual genome assembly point of view, the output results were not optimal, in comparison to the other assemblers on other platforms such as Galaxy and HIVE, and here is why -- 

1. Based on the assembly report, it generated 631 contigs, and the longest one is 12809 nucleotides, which is ~43% of the reference (18965/29903), which by far is the lowest across all other assemblers on other platforms;

<img width="1155" alt="Screen Shot 2023-01-25 at 3 14 05 PM" src="https://user-images.githubusercontent.com/61590108/214679517-45c6c258-3b2b-4387-8cd6-59911b5580cc.png">

2. In the sequence view window in the bottom: it is not the most optimal place to edit sequence; MEGA is better at accomplishing the;

3. Geneious does not have the capacity of processing large files. In particular, when a large file is uploaded, it takes days to process which is not a sustainable way moving forward. 

In conclusion, after speaking with both Dr. Mazumder and Millicent, we all agreed that Geneious is just like all other assemblers that have pros and cons; however, it is not our No.1 Platform to choose when in regards to performing genome assembly.
